examples/libs/tezos/basic-tx/taquito-injection/main.ts

```diff
@@ -0,0 +1,78 @@
+import { DfnsWallet } from '@dfns/lib-taquito'
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+import { RpcClient} from '@taquito/rpc'
+import { LocalForger } from '@taquito/local-forging';
+import { TezosToolkit } from '@taquito/taquito'
+
+import * as dotenv from 'dotenv';
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+    credId: process.env.DFNS_CRED_ID!,
+    appOrigin: process.env.DFNS_APP_ORIGIN!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({
+    walletId: walletId,
+    dfnsClient,
+    maxRetries: 10,
+  })
+}
+
+async function main() {
+  // Replace with the RPC url/ chainId you want
+  const client = new RpcClient('https://ghostnet.ecadinfra.com', 'main') 
+
+  const senderWallet = await initDfnsWallet(process.env.DFNS_WALLET_ID!)
+  console.log("Tezos sender address: %s", senderWallet.address)
+
+  const Tezos = new TezosToolkit(client)
+
+  const forger = new LocalForger()
+
+  Tezos.setForgerProvider(forger)
+
+  Tezos.setProvider({ signer: senderWallet })
+
+  const receiverAddress = 'tz2JZdBTEkAYXp53X4r5xW5dQ845zi9Z28mT'
+
+  const estimate = await Tezos.estimate.transfer({
+    source: senderWallet.address,
+    to: receiverAddress,
+    amount: 1,
+  })
+
+  const prepared = await Tezos.prepare.transaction({
+    source: senderWallet.address,
+    to: receiverAddress,
+    amount: 1,
+    fee: estimate.suggestedFeeMutez,
+    gasLimit: estimate.gasLimit,
+    storageLimit: estimate.storageLimit,
+  })
+
+  const forgeable = await Tezos.prepare.toForge(prepared)
+
+  console.log(`Operation prepared: ${JSON.stringify(forgeable, null, 2)}`)
+
+  const forgedBytes = await forger.forge(forgeable)
+
+  const signedOp = await Tezos.signer.sign(forgedBytes, new Uint8Array([3]))
+
+  const op = await Tezos.rpc.injectOperation(signedOp.sbytes)
```

Comment:
the example looks overly complicated. this is how internally things work. for the purpose of these examples, we want to showcase our integration is making the development as simple as possible. looking at the doc, if everything is correctly wired up, we should be able to just do something like

```
await Tezos.contract.transfer({ to: 'tz2JZdBTEkAYXp53X4r5xW5dQ845zi9Z28mT', amount: 1 });
```